scripts/code_size_compare.py

```diff
@@ -32,11 +32,62 @@
 
 from mbedtls_dev import build_tree
 
+class Size:
+    """Represent code size; split into text, data, and bss"""
+    def __init__(self, text: int, data: int, bss: int, total: int):
+        self.text = text
+        self.data = data
+        self.bss = bss
+        self.total = total
+    def __eq__(self, __o) -> bool:
```

Comment:
While we don't use them now, I'd like to keep them in simply because they allow a `Size` object to be operated on like a number. I added them for completeness, and this makes future use of the class easier.